.github/ISSUE_TEMPLATE/false_negative.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug Report (False Negative)
+about: Create a bug report about missing lint warnings in an existing lint
+labels: L-bug, L-false-negative
+---
+<!--
+Thank you for filing a bug report! 🐛 Please provide a short summary of the bug,
+along with any information you feel relevant to replicating the bug.
+-->
+Lint name:
+
+
+I tried this code:
+
+```rust
+<code>
+```
+
+I expected to see this happen: *explanation*
+
+Instead, this happened: *explanation*
+
+### Meta
+
+- `cargo clippy -V`: e.g. clippy 0.0.212 (f455e46 2020-06-20)
```

Comment:
Unrelated thought: Should we remove the `0.0.212` part, when printing the Clippy version?

Reply:
We could adopt a version scheme that follows rustc versions (similar to what cargo does).
nightly would be clippy 0.1.50, and stable clipy 0.1.49 then.

Reply:
created https://github.com/rust-lang/rust-clippy/issues/6499 for the version discussion